src/progressbar/progressbar.js

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import * as React from 'react';
+import {getOverrides} from '../helpers/overrides';
+import {
+  Root as StyledRoot,
+  Bar as StyledBar,
+  Label as StyledLabel,
+} from './styled-components';
+
+import type {ProgressBarPropsT} from './types';
+
+class ProgressBar extends React.Component<ProgressBarPropsT, {}> {
+  static defaultProps = {
+    getProgressLabel: (value: number, successValue: number) =>
+      `${Math.round((value / successValue) * 100)}% Loaded`,
+    successValue: 100,
+    value: 0,
+    overrides: {},
+  };
+
+  render() {
+    const {overrides = {}, getProgressLabel, value, successValue} = this.props;
+    const [Root, rootProps] = getOverrides(overrides.Root, StyledRoot);
+    const [Bar, barProps] = getOverrides(overrides.Bar, StyledBar);
+    const [Label, labelProps] = getOverrides(overrides.Label, StyledLabel);
+    const sharedProps = {
+      $value: value,
+      $successValue: successValue,
+    };
+    return (
+      <Root role="progressbar" {...sharedProps} {...rootProps}>
+        <Bar {...sharedProps} {...barProps} />
```

Comment:
Most other sliders (including native) allow clicking on the bar to set the value.

![kapture 2018-10-09 at 10 02 42](https://user-images.githubusercontent.com/875591/46685517-87525980-cbaa-11e8-942c-8ec76d046081.gif)

Do we plan to support that?